src/main/java/Bank.java

```diff
@@ -0,0 +1,28 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Bank {
+    static void creditBank(int credit) throws IOException {
+        int result;
+
+        //result = credit - repayment;
+        if (credit > 0){
+            BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+            int repayment = Integer.parseInt(reader.readLine());
+            System.out.println(credit - repayment +" - Ваша задолженность" );
```

Comment:
![image](https://user-images.githubusercontent.com/39908109/102105796-7eb7d680-3e38-11eb-8667-c4c9f7fa5510.png)

Reply:
Fix case,when user pays more than expected. I think it should display only the overpaid amount.